packages/spacecat-shared-data-access/src/service/import-url/accessPatterns.js

```diff
@@ -96,3 +96,23 @@ export const getImportUrlsByJobIdAndStatus = async (dynamoClient, config, log, j
   });
   return items.map((item) => ImportUrlDto.fromDynamoItem(item));
 };
+
+/**
+ * Get Import Urls by Job ID
+ * @param {DynamoClient} dynamoClient
+ * @param {Object} config
+ * @param {Logger} log
+ * @param {string} jobId
+ * @returns {Promise<ImportUrlDto[]>}
+ */
+export const getImportUrlsByJobId = async (dynamoClient, config, log, jobId) => {
+  const items = await dynamoClient.query({
+    TableName: config.tableNameImportUrls,
+    IndexName: config.indexNameImportUrlsByJobIdAndStatus,
```

Comment:
Can we reuse the `indexNameImportUrlsByJobIdAndStatus` for this? I thought that a new GSI would be needed.

Reply:
Yes. We can. The partition key is `jobId` - it will work for this query as well.